guides/common/attributes-katello.adoc

```diff
@@ -5,3 +5,49 @@
 :installer-log-file: /var/log/foreman-installer/katello.log
 :project-installation-guide-title: Installing Foreman {ProjectVersion} server with Katello {KatelloVersion} plugin on Enterprise Linux
 :smartproxy_port: 9090
+
+//Added by Marie for GA
```

Comment:
I don't think you need to add these to katello since they're either unused or should be in `-base`.

Reply:
Oh, I've added them because the original file used in 2.5 includes statement that these repo attributes are relevant for Satellite and Katello.